advisories/hackage/cryptonite/HSEC-2025-0002.md

```diff
@@ -0,0 +1,59 @@
+```toml
+[advisory]
+id = "HSEC-2025-0002"
+cwe = []
+keywords = ["crypto", "historical"]
```

Comment:
I think we should no include `historical` tag here, because the issue is in current releases.

Reply:
Thanks, I've dropped it